src/libs/ajax/DatasetBuilder.ts

```diff
@@ -1,11 +1,60 @@
+import { DomainType, ProgramDataType } from 'src/pages/library/datasetBuilder/dataset-builder-types';
```

Comment:
the types describing things coming from the api payload should live here near the data-access area, not in src/pages/...

Reply:
see ui stack section here:

https://docs.google.com/document/d/1vefkF3tbIPgn_2Kbx6lscFK6nYoZoaowlkXXkszxijA/edit?pli=1#heading=h.w7o97nwl0rth

Reply:
I'll move the types that are API specific into the Ajax code. I had planned to do that but forgot to take care of it.